platform/ios/src/MGLAnnotationView.h

```diff
@@ -26,6 +26,14 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, nullable) NSString *reuseIdentifier;
 
+/**
+ Annotation view is centered at the coordinate point of the associated annotation.
+ 
+ By changing this property you can reposition the view as needed. The offset is measured in points.
+ Positive offset moves the annotation view towards the bottom right, while negative offset moves it towards the top left.
+ */
+@property (nonatomic) CGPoint centerOffset;
```

Comment:
CGPoint is fine, but [CGVector](https://developer.apple.com/library/ios/documentation/GraphicsImaging/Reference/CGGeometry/index.html#//apple_ref/c/tdef/CGVector) would be better since we only support iOS 7 and above.